systemd/renderd.service

```diff
@@ -1,8 +1,12 @@
 [Unit]
-Description=Rendering daemon for Openstreetmap tiles
+Description=Daemon that renders map tiles using mapnik
+Documentation=man:renderd
+After=network.target auditd.service
 
 [Service]
-ExecStart=/usr/bin/renderd -f
+RuntimeDirectory=/run/renderd
```

Comment:
/run/ isn't needed here

Reply:
Also if this is referenced to in the [configuration example](https://github.com/openstreetmap/mod_tile/blob/a3f4230df6bc320b3b564ab20a29b57f787dbfe4/renderd.conf#L4) and [code](https://github.com/openstreetmap/mod_tile/blob/a3f4230df6bc320b3b564ab20a29b57f787dbfe4/includes/protocol.h#L37)?

Reply:
Yeah, systemd adds `/run/` to `RuntimeDirectory` and `/var/cache/` to `CacheDirectory` automatically